requirements.txt

```diff
@@ -0,0 +1 @@
+threat_intel
```

Comment:
I was under the impression that we needed threat_intel to run the collector, that's why I created a different requirement file (for running, not dev). But you are absolutely correct, so I'm moving it to -dev.